cli/azd/pkg/environment/environment.go

```diff
@@ -202,3 +202,14 @@ func (e *Environment) SetPrincipalId(principalID string) {
 func (e *Environment) GetPrincipalId() string {
 	return e.Values[PrincipalIdEnvVarName]
 }
+
+// Creates a slice of key value pairs like `KEY=VALUE` that
+// can be used to pass into command runner or similar constructs
+func (e *Environment) Environ() []string {
```

Comment:
Was surprised to not see this already existing. Looks like we created a private implementation for `framework_service_npm.go`. Mind updating that to reference this?